### PR TITLE
Add option to ignore users per Role after logout

### PR DESCRIPTION
**Pull request by urosevic**
Implement new option **Ignore users after logout** on **General** tab which drop cookie **yst_ignore_user** if user belong to role selected for **Ignore users** option.

Then when user logout from WordPress, he'll keep cookie in browser. Inline scripts tracking-ga-js.php and tracking-universal.php contains condition to check does cookie exists and if so, simply skip execution of GA tracking code to prevent tracking.

This feature is related to my feature request posted to [WP community forum](https://wordpress.org/support/topic/programatically-enable-debug-mode).

### CODE REVIEW

**Comment by Rarst** [comment]
Please fix the coding style, as reported by Travis, for PR to be considered. :)

**Comment by omarreiss** [comment]
@urosevic We've reviewed this functionality quite thoroughly and came to the conclusion that we don't want to add it. There are a couple of reasons for it. 

First of all, this PR is incomplete since it doesn't address what to do with click tracking of outbound links. The onclick events that are added will refer to a non-existing object, causing JS errors on the frontend which might break other things as well.

Second, when we try to devise a solution for this we encounter another problem. Since page caching is disabled for logged in users, their frontend visits will not affect the page cache. Changing the output on the frontend is not a problem then. But for non logged-in users this is another story. Two problems might occur:
1) non-logged in users might be served a page cache with tracking enabled
2) non-logged in users might trigger a page without tracking to be stored as page cache, causing all visits to those page to be untracked.

Third, trying to solve this with a cookie is really not that reliable and serves an edgecase that seems only really interesting from a testing perspective. This fix doesn't really have a good answer to what will happen when a user's capability changes or when the ignored user roles are updated. 

Bottom line: the behavior introduced here serves too much of an edgecase and possibly has too many possible risky/negative caveats.
